include/triton/Dialect/TritonGPU/Transforms/Utility.h

```diff
@@ -193,6 +193,8 @@ bool isPureUnaryInlineAsm(Operation *op);
 // read the compute capability from the module attributes
 int getNVIDIAComputeCapability(Operation *module);
 
+// Convert \param op operands and results to layout \param encoding.
+void convertOpEncoding(Attribute encoding, Operation *op);
```

Comment:
I moved (and renamed) this function from Coalece.cpp so I could use it in BypassLDS pass since I needed this exact functionality.